credits.php

```diff
@@ -0,0 +1,197 @@
+                    </div>
+                    <div class="col-md-2">
+                        <div class="form-group">
+                            <label>Date To</label>
+                            <input type="date" class="form-control" name="dtt" max="2999-12-31"
+                                value="<?php echo nullable_htmlentities($dtt); ?>">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </form>
+        <hr>
+        <div class="table-responsive-sm">
+            <table class="table table-striped table-borderless table-hover">
+                <thead class="text-dark <?php if ($num_rows == 0) { echo "d-none"; } ?>">
+                    <tr>
+                        <th><a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_client&order=<?php echo $disp; ?>">Client Name</a></th>
+                        <th><a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_account&order=<?php echo $disp; ?>">Account
+                                Name</a></th>
+                        <th class="text-right
+                            <?php if ($sort == "credit_amount") { echo "sorting-$order"; } ?>">
+                            <a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_amount&order=<?php echo $disp; ?>">Amount</a>
+                        </th>
+                        <th><a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_date&order=<?php echo $disp; ?>">Date</a>
+                        </th>
+                        <th><a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_reference&order=<?php echo $disp; ?>">Reference</a>
+                        </th>
+                        <th><a class="text-dark"
+                                href="?<?php echo $url_query_strings_sort; ?>&sort=credit_payment&order=<?php echo $disp; ?>">Origin</a>
+                        </th>
+
+                        <th>Actions</th>
+
+                    </tr>
+                </thead>
+                <tbody>
+                <?php
+                    while ($row = mysqli_fetch_array($sql)) {
+                        $credit_id = $row['credit_id'];
+                        $credit_amount = $row['credit_amount'];
+                        $credit_currency_code = $row['credit_currency_code'];
+                        $credit_date = $row['credit_date'];
+                        $credit_reference = $row['credit_reference'];
+                        $credit_client_id = $row['credit_client_id'];
+                        $credit_payment_id = $row['credit_payment_id'];
+                        $credit_account_id = $row['credit_account_id'];
+                        $client_name = $row['client_name'];
```

Comment:
@o-psi These will have be sanitized for output

Reply:
Done